code/modules/admin/verbs/randomverbs.dm

```diff
@@ -513,6 +513,81 @@ Traitors and the like can also be revived with the previous role mostly intact.
 
 	return new_character
 
+/client/proc/spawn_character_mob()

```

Comment:
This should probably be exported to randomverbs_vr.dm

Reply:
Oh right you fella's still have that whole downstream thing going on.
Yeah aight.